R/report.R

```diff
@@ -1,153 +1,154 @@
 
 create_report <- function(input, output_dir, has_code, opts) {
+  report <- ""
+  if (is.null(opts$suppress_reports) || isFALSE(opts$suppress_reports)) {
```

Comment:
Instead of indenting the majority of the function, could you instead use an early return, e.g.:

```
if (opts$suppress_reports) {
  return("")
}
```